CHANGELOG.md

```diff
@@ -4,6 +4,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.0.1] - 2019-06-11
+### Changed
+- Upgrade `xhr-mock` to `2.4.0`.
```

Comment:
I think `changed` indicates changes in existing functionality.
Is upgrading libraries suitable for that?
https://keepachangelog.com/en/1.0.0/

Reply:
Hmm `changed` felt natural to me: "we changed a direct dependency". But I agree, it's not functionality. 

How about this?
For `xhr-mock`: Fixed: stopped using preview version.
For `axios`: Security: upgraded version to fix vulnerability.

Reply:
> For xhr-mock: Fixed: stopped using preview version.

I wonder if it is really necessary to describe changelogs for all dependencies every time.
In this case, especially, it's included in `devDependencies` which is not related to the users.
What do you think?

> For axios: Security: upgraded version to fix vulnerability.

I agree!

Reply:
> I wonder if it is really necessary to describe changelogs for all dependencies every time.
In this case, especially, it's included in devDependencies which is not related to the users.
What do you think?

Hm effectively the user will not be impacted by this change.
-> Let's skip changes that don't affect the user.
-> I'll remove the mention about `xhr-mock`

Reply:
Ah, if it's all about user impact, how about we only mention this:

```
### Security
- Upgrade dependencies to fix vulnerabilities.
```

No other entries.
Is it OK to proceed with that?

Reply:
I think it looks good!